examples/acados_python/zoRO_example/diff_drive/diff_drive_utils.py

```diff
@@ -51,8 +52,11 @@ def get_latex_plot_params():
 
 def plot_timings(timing_dict):
     # latexify plot
-    params = get_latex_plot_params()
-    matplotlib.rcParams.update(params)
+    if shutil.which('latex'):
+        params = get_latex_plot_params()
+        matplotlib.rcParams.update(params)
```

Comment:
I really like to have this check!
Can you just implement it within the `latexify` function itself?